### PR TITLE
[TECH-1426] Feat/hide-feed-on-subscriptions

### PR DESCRIPTION
**Pull request by TheHussein**
N.B.: Linked to https://github.com/c-commerce/charles-agent-ui/pull/1154/ and https://github.com/c-commerce/charles-client-api/pull/536

### CODE REVIEW

File: src/universe/index.ts [hunk 1]
```diff
@@ -580,8 +580,7 @@ export class Universe extends APICarrier {
       universeTopics.api.feedsActivities.generateTopic(),
       universeTopics.api.feedsMessages.generateTopic(),
       universeTopics.api.feedsEvents.generateTopic(),
-      universeTopics.api.people.generateTopic()
-    ]
+      universeTopics.api.people.generateTopic()]
```
Comment (eljefedelrodeodeljefe) [inline src/universe/index.ts:583]: this should a linter error actually

File: src/universe/index.ts [hunk 2]
```diff
@@ -626,12 +626,15 @@ export class Universe extends APICarrier {
...
+      let feedStruct
       if ((msg as realtime.RealtimeMessageMessage).payload.event) {
         const feedPayload: FeedRawPayload = { id: (msg as realtime.RealtimeMessageMessage).payload.event.feed }
         feed = Feed.create(feedPayload, this, this.http, this.mqtt)
+        // This one is an actual feed, the one above is just a fake object with just an id
+        feedStruct = (msg as realtime.RealtimeMessageMessage).payload.feed
         event = Event.create((msg as realtime.RealtimeMessageMessage).payload.event as EventRawPayload, feed, this, this.http)
       }
-      this.emit('universe:feeds:events', { ...msg, event, feed: feed })
+      this.emit('universe:feeds:events', { ...msg, event, feed: feed, feedStruct })
```
Comment (eljefedelrodeodeljefe) [inline src/universe/index.ts:637]: Omar did not convince to have it like this, but has good reasoning to keep it that way. Essentially, this change will ensure being backwards compatible, and the actual. offender of quirk is the original author of the above (likely me).

**Review by eljefedelrodeodeljefe — CHANGES_REQUESTED** [review]
One nit to address, otherwise LGTM with a comment

**Comment by TheHussein** [comment]
Superseded by: https://github.com/c-commerce/charles-browser-sdk/pull/146

**Pull request closed without merging**